graphqlapi/src/main/java/io/stargate/graphql/schema/DmlSchemaBuilder.java

```diff
@@ -369,27 +381,35 @@ private GraphQLType buildQueryOptionsInputType() {
   private GraphQLType buildOrderType(Table table) {
     GraphQLEnumType.Builder input =
         GraphQLEnumType.newEnum().name(nameMapping.getGraphqlName(table) + "Order");
-    for (Column columnMetadata : table.columns()) {
-      input.value(nameMapping.getGraphqlName(table, columnMetadata) + "_DESC");
-      input.value(nameMapping.getGraphqlName(table, columnMetadata) + "_ASC");
+    for (Column column : table.columns()) {
+      String graphqlName = nameMapping.getGraphqlName(table, column);
+      if (graphqlName != null) {
+        input.value(graphqlName + "_DESC");
+        input.value(graphqlName + "_ASC");
+      }
     }
     return input.build();
   }
 
   private GraphQLType buildInputType(Table table) {
     GraphQLInputObjectType.Builder input =
         GraphQLInputObjectType.newInputObject().name(nameMapping.getGraphqlName(table) + "Input");
-    for (Column columnMetadata : table.columns()) {
-      try {
-        GraphQLInputObjectField field =
-            GraphQLInputObjectField.newInputObjectField()
-                .name(nameMapping.getGraphqlName(table, columnMetadata))
-                .type(fieldInputTypes.get(columnMetadata.type()))
-                .build();
-        input.field(field);
-      } catch (Exception e) {
-        log.error(
```

Comment:
We don't at least want a warn on the server side too?

Reply:
Right, I'll add them back.

I think error logs should target operators, so I'm always wary about logging too much: for example if the CQL data model causes a a name clash in `NameMapping`, it's a user issue, not something an operator should be paged about.
But the catch blocks in this class cover unexpected errors, so it's reasonable to log at WARN level.